jvb/src/main/kotlin/org/jitsi/videobridge/cc/allocation/PacketHandler.kt

```diff
@@ -166,7 +168,7 @@ internal class PacketHandler(
         }
     }
 
-    fun timeSinceFirstMedia(): Long = if (firstMediaMs == -1L) -1 else clock.instant().toEpochMilli() - firstMediaMs
+    fun timeSinceFirstMedia(): Duration = firstMedia?.let { Duration.between(it, clock.instant()) } ?: Duration.ZERO
```

Comment:
I'm not crazy about using `Duration.ZERO` as "never", though I guess for how we're using it it doesn't matter.